ui/components/MesheryPatterns.js

```diff
@@ -809,25 +874,58 @@ function MesheryPatterns({
 
   const handleDeploy = (pattern_file, pattern_id, name) => {
     updateProgress({ showProgress: true });
-    dataFetch(
-      ctxUrl(DEPLOY_URL, selectedK8sContexts),
-      {
-        credentials: 'include',
-        method: 'POST',
-        body: JSON.stringify({
-          pattern_file: pattern_file,
-          pattern_id: pattern_id,
-        }),
+
+    //NOTE: needed to hardcode 'pattern/deploy' route here since for some reason I kept getting errors because
```

Comment:
Thanks for the note @cesartheroman , i don't think so that rtk query add the endpoint to ctxurl since its a function that takes a url to trigger. You can propose any change or workaround here because using rtk query like this to take url as hardcoded doesn't give advantage of redux